test/chrono-test.cc

```diff
@@ -121,6 +121,11 @@ TEST(chrono_test, format_tm) {
       make_tm(2000, 1, 2, 12, 14, 16),    // W52
       make_tm(2000, 1, 3, 12, 14, 16)     // W1
   };
+
+#if defined(__MINGW32__) && !defined(_UCRT)
+  GTEST_SKIP();
```

Comment:
Could you add a comment clarifying why we skip this test?

Reply:
Done,